seacrowd/sea_datasets/udhr_lid/udhr_lid.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+import pandas as pd
+
+_CITATION = """\
+@inproceedings{
+  kargaran2023glotlid,
+  title={{GlotLID}: Language Identification for Low-Resource Languages},
+  author={Kargaran, Amir Hossein and Imani, Ayyoob and Yvon, Fran{\c{c}}ois and Sch{\"u}tze, Hinrich},
+  booktitle={The 2023 Conference on Empirical Methods in Natural Language Processing},
+  year={2023},
+  url={https://openreview.net/forum?id=dl4e3EBz5j}
+}
+"""
+
+_LANGUAGES = ["sun", "ace", "mad", "lao", "cfm", "hnj", "min", "zlm", "tha", "blt", "hni", "jav", "tdt", "cnh", "khm", "ban", "ind", "mya", "ccp", "duu", "tet", "kkh", "bug", "vie"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = False
+
+_DATASETNAME = "udhr_lid"
+
+_DESCRIPTION = """\
+The UDHR-LID dataset is a refined version of the Universal Declaration of Human Rights, tailored for language identification tasks. 
+It removes filler texts, repeated phrases, and inaccuracies from the original UDHR, focusing only on cleaned paragraphs. 
+Each entry in the dataset is associated with a specific language, providing long, linguistically rich content. 
+This dataset is particularly useful for non-parallel, language-specific text analysis in natural language processing.
```

Comment:
Trailing whitespaces

```suggestion
The UDHR-LID dataset is a refined version of the Universal Declaration of Human Rights, tailored for language identification tasks.
It removes filler texts, repeated phrases, and inaccuracies from the original UDHR, focusing only on cleaned paragraphs.
Each entry in the dataset is associated with a specific language, providing long, linguistically rich content.
This dataset is particularly useful for non-parallel, language-specific text analysis in natural language processing.
```